docs/get-box/1-introduction.md

```diff
@@ -8,7 +8,7 @@ order: 1
 
 # Introducing Get Box
 
-Jangala's Get Box is designed to provide effective internet access for households and small groups. It's designed to be easy to set up, provide robust performance, and make the most of available data [^1].
+Jangala's Get Box was designed to provide effective internet access for households. It's designed to be easy to set up, provide robust performance, and make the most of available data [^1].
```

Comment:
It would be good to get consistency on whether we talk of the design of GB in present or past tense. The trend in products is to use present tense, see [this](https://www.google.com/search?q=%22macbook+pro+is+designed%22) vs [this](https://www.google.com/search?q=%22macbook+pro+was+designed%22)

Reply:
Would like @Ljangala's input into this :)